### PR TITLE
Mar23/configuring update

### PR DESCRIPTION
**Pull request by OliviaShoup**
This PR updates **Configuring Clusters** for the new Slurm Settings + changes to General Settings, Partition Settings, and settings for existing clusters.

### CODE REVIEW

File: docs/configuring-clusters.md [hunk 1]
```diff
@@ -267,75 +326,88 @@ Please note that GPU and spot instances cannot be live migrated. When supported,
...
+### General Settings for Existing Clusters 
 
-If you want to set up an NFS, please contact us or your PW platform administrator.
+#### <Highlight color="#ff5800">Resource Account</Highlight>
 
-#### <Highlight color="#ff5800">NFS Size</Highlight>
+Use this dropdown menu to select the account that your organization uses for a specific cloud service provider.
 
-Use this field to enter the size of your NFS.
+By default, this menu will show the resource account that your organization has selected for that type of cluster (for example, the screenshot above shows a Google cluster, and **Resource Account** was automatically populated with the _Pworks GCP_ account).
```
Comment (Egr711) [inline docs/configuring-clusters.md:341]: screenshot shows existing cluster instead of google cluster, could remove this or rephrase this to fit in with Existing Cluster type

**Review by giraffesyo — APPROVED** [review]
